### DISCUSSION TITLE
Override ComboBox AutoComplete rules

### DISCUSSION THREAD

**Discussion by fleixi** [picoe/Eto#1980] [Q&A] [answered]
Whats the best way to define new rules for the AutoComplete function of the combobox? I have a list of names and i like to have a better search rule. At the moment i need to input the first letters but i like to search for the lastname as well.

At the moment i only see to use the OnTextChanged and change the text based on the input, but this looks like a over complex way with many disadvantage.

**Comment by cwensley** [comment, marked as answer]
Unfortunately there's no real way to override the default behaviour, some platforms don't allow it.  One way people have done their own autocomplete is by creating a borderless form and showing it below a TextBox or ComboBox with the appropriate content. One advantage to this approach is that the content of the drop down can be customized to whatever you like.
